gradle/libs.versions.toml

```
@@ -109,9 +110,24 @@ lucene-sandbox = { group = "org.apache.lucene", name = "lucene-sandbox", version
 lucene-spatial-extras = { group = "org.apache.lucene", name = "lucene-spatial-extras", version.ref = "lucene" }
 lucene-spatial3d = { group = "org.apache.lucene", name = "lucene-spatial3d", version.ref = "lucene" }
 lucene-suggest = { group = "org.apache.lucene", name = "lucene-suggest", version.ref = "lucene" }
+netty-buffer = { group = "io.netty", name = "netty-buffer", version.ref = "netty" }
+netty-codec = { group = "io.netty", name = "netty-codec", version.ref = "netty" }
+netty-codec-dns = { group = "io.netty", name = "netty-codec-dns", version.ref = "netty" }
+netty-codec-http = { group = "io.netty", name = "netty-codec-http", version.ref = "netty" }
+netty-codec-http2 = { group = "io.netty", name = "netty-codec-http2", version.ref = "netty" }
+netty-common = { group = "io.netty", name = "netty-common", version.ref = "netty" }
+netty-handler = { group = "io.netty", name = "netty-handler", version.ref = "netty" }
+netty-resolver-dns = { group = "io.netty", name = "netty-resolver-dns", version.ref = "netty" }
+netty-resolver = { group = "io.netty", name = "netty-resolver", version.ref = "netty" }
+netty-transport = { group = "io.netty", name = "netty-transport", version.ref = "netty" }
+netty-transport-native-unix-common = { group = "io.netty", name = "netty-transport-native-unix-common", version.ref = "netty" }
 protobuf = { group = "com.google.protobuf", name = "protobuf-java", version.ref = "protobuf" }
-reactivestreams = { group = "io.projectreactor", name = "reactor-core", version.ref = "reactor" }
-reactorcore = { group = "org.reactivestreams", name = "reactive-streams", version.ref = "reactivestreams" }
+reactivestreams = { group = "org.reactivestreams", name = "reactive-streams", version.ref = "reactivestreams" }
+reactorcore = { group = "io.projectreactor", name = "reactor-core", version.ref = "reactor" }
```

To be consistent with all other `reactor-xxx` dependencies?

```suggestion
reactor-core = { group = "io.projectreactor", name = "reactor-core", version.ref = "reactor" }
```

---

Found the syntax here: https://github.com/gradle/gradle/issues/18034#issuecomment-899594513

I can add hyphens in here, but hyphens because `.` when referencing them in a `build.gradle` file.

i.e.

```
api libs.reactor.core
```

---

> Found the syntax here: [gradle/gradle#18034 (comment)](https://github.com/gradle/gradle/issues/18034#issuecomment-899594513)
> 
> I can add hyphens in here, but hyphens because `.` when referencing them in a `build.gradle` file.
> 
> i.e.
> 
> ```
> api libs.reactor.core
> ```

I think this is fine, right? Since we have all other artifacts named with hyphens (reactor-netty-http, netty-common, ...)